book/_toc.yml

```diff
@@ -24,6 +24,10 @@ parts:
         - file: tutorials/cryocloud_demo/CryoCloud_demo.ipynb
         - file: tutorials/nb-to-package/index
           sections:
+            - file: tutorials/nb-to-package/OOP_intro
+              sections:
+              - file: tutorials/nb-to-package/climate_model.py
+              - file: tutorials/nb-to-package/co2_emissions.py
```

Comment:
```suggestion
```
(I don't think scripts will render), you can leave them out in the toc, or perhaps also show them in a code block in your OOP_intro markdown file